dbt-cta/spoke_mo/models/1_cta_incremental/_1_cta_incremental__models.yml

```diff
@@ -0,0 +1,1131 @@
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_tag_canned_response_hashid
+        description: ''
+        tests:
+            - not_null
+            - unique
+
+  - name: zip_code_base
+    description: ''
+    columns:
+      - name: zip
+        description: ''
+        tests:
+            - not_null
+            - unique
+      - name: city
+        description: ''
+      - name: latitude
+        description: ''
+      - name: timezone_offset
+        description: ''
+      - name: state
+        description: ''
+      - name: has_dst
+        description: ''
+      - name: longitude
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_zip_code_hashid
+        description: ''
+        tests:
+            - not_null
+            - unique
+
+  - name: opt_out_base
+    description: ''
+    columns:
+      - name: reason_code
+        description: ''
+      - name: assignment_id
+        description: ''
+      - name: organization_id
+        description: ''
+      - name: created_at
+        description: ''
+      - name: id
+        description: ''
+        tests:
+            - not_null
+            - unique
+      - name: cell
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_opt_out_hashid
+        description: ''
+        tests:
+            - not_null
+            - unique 
+  - name: user_cell_base
+    description: ''
+    columns:
+      - name: is_primary
+        description: ''
+      - name: user_id
+        description: ''
+      - name: service
+        description: ''
+      - name: id
+        description: ''
+        tests:
+            - not_null
+            - unique
+        tests:
```

Comment:
looks like some dupe tests for this column here

Reply:
thanks! fixed and will merge now.

Reply:
oops, since i changed something you reviewed i have to re-request approval @jitoquinto